### DISCUSSION TITLE
Issue with Heatwave Buildtools namespace

### DISCUSSION THREAD

**Discussion by miguelmartins1987** [wixtoolset/issues#8086] [Questions] [answered]
I'm trying to harvest a certain folder via the FireGiant.HeatWave.BuildTools.wixext extension. Something like this:

```xml
<Wix xmlns="http://wixtoolset.org/schemas/v4/wxs" xmlns:hbt="http://www.firegiant.com/schemas/v4/wxs/heatwave/buildtools">
	<Package Name="..." Manufacturer="..." Version="1.0.0.0" UpgradeCode="...">
		<MajorUpgrade DowngradeErrorMessage="!(loc.DowngradeError)" />
		<MediaTemplate EmbedCab="yes" />
		<hbt:HarvestFolder Source="$(var.myProject.TargetDir)"/>
(...)
```

I've noticed, however... The namespace I'm referring to as **xmlns:hbt**... Its URL seems to be unavailable (i.e. accessing http://www.firegiant.com/schemas/v4/wxs/heatwave/buildtools seems to yield a 404 page).

Does this mean the extension is currently unusable? And if so, is there a workaround for harvesting an entire folder?

**Comment by robmen** [comment, marked as answer]
No. XML namespaces are just names. They don't have to be a URL (although they often are) and the URL does not have to resolve to a web page (although sometimes it does).

Note: the `HarvestFolder` element is (currently) expected to be under a `Component` element. See the [Advanced Harvesting documentation](https://www.firegiant.com/docs/heatwave-build-tools/harvesting/) for more details.
